std/algorithm/sorting.d

```diff
@@ -130,55 +132,89 @@ unittest
     assert(b == [ 3, 4, 5, 6 ]);
 }
 
-// isSorted
-/**
-Checks whether a forward range is sorted according to the comparison
-operation $(D less). Performs $(BIGOH r.length) evaluations of $(D
-less).
-
-Params:
-    less = Predicate the range should be sorted by.
-    r = Forward range to check for sortedness.
-
-Returns: true if the range is sorted, false otherwise.
-*/
-bool isSorted(alias less = "a < b", Range)(Range r) if (isForwardRange!(Range))
+private bool isSortedImpl(alias less = "a < b", bool isStrict = false, Range)(Range r)
+if (isForwardRange!(Range))
```

Comment:
I know this is from the old code, but we now support the shortcut syntax `isForwardRange!Range`